src/ContainerCommitModal.jsx

```diff
@@ -0,0 +1,176 @@
+                            <td align="right"><label className="control-label">{_("Tag")}</label></td>
+                            <td colSpan="3">
+                                <input name="tag" className="form-control-commit container-tag" type="text" onChange={this.handleInputChange} required="true" />
+                            </td>
+                        </tr>
+                        <tr>
+                            <td align="right"><label className="control-label">{_("Author")}</label></td>
+                            <td colSpan="3">
+                                <input name="author" className="form-control-commit container-author" type="text" onChange={this.handleInputChange} />
+                            </td>
+                        </tr>
+                        <tr>
+                            <td align="right"><label className="control-label">{_("Message")}</label></td>
+                            <td colSpan="3">
+                                <input name="message" className="form-control-commit container-message" type="text" onChange={this.handleInputChange} />
+                            </td>
+                        </tr>
+                        <tr>
+                            <td align="right"><label className="control-label">{_("Command")}</label></td>
+                            <td colSpan="3">
+                                <input name="command" className="form-control-commit container-command" type="text" defaultValue={this.props.containerWillCommit.Config ? this.props.containerWillCommit.Config.Cmd.join(" ") : ""} onChange={this.handleInputChange} />
+                            </td>
+                        </tr>
+                        <tr>
+                            <td align="right"><label className="control-label">{_("Pause")}</label></td>
+                            <td colSpan="3">
+                                <label>
+                                    <input name="pause" className="container-pause" type="checkbox" defaultChecked onChange={this.handleInputChange} />
+                                    <span>{_("pause the container")}</span>
+                                </label>
+                                <div className="containers-run-inline" />
+                            </td>
+                        </tr>
+                        <tr>
+                            <td valign="top" align="right"><label className="control-label">{_("On Build")}</label></td>
+                            <td colSpan="3">
+                                <label>
+                                    <input name="setonbuild" className="container-label" type="checkbox" disabled={this.state.onbuildDisabled} onChange={this.handleInputChange} />
+                                    <span>{_("Set container on build variables")}</span>
+                                </label>
+                                {(this.state.setonbuild && <div>{onbuilds}</div>) }
```

Comment:
It's a bit weird that you have implemented the build variables forms, but we can't actually see it at this point in the UI, since the handleInputChanged for the checkbox is not implemented.
I think that it makes sense to implement at a certain point only what can be actually reached in UI and not more.